case/step/case_model.py

```
@@ -0,0 +1,559 @@
+
+
+def add_usb_chamfer(part):
+    """Bottom-only USB lead-ins (toward the open bottom) -- see _usb_bottom_bevel.
+    No clip needed: below each window is clear wall down to the open bottom.  face_pos
+    values are the MEASURED inner wall faces at each window (right side, mine frame)."""
+    try:
+        C = USB_CHAMFER
+        b1 = _usb_bottom_bevel('x', +1, -86.65, 6.9, C, 'y', 1.0, 13.7)     # side wall
+        b0 = _usb_bottom_bevel('y', -1, 62.29, 6.9, C, 'x', -71.1, -58.5)   # back wall
+        return part - b1 - b0
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)